src/Cake.Common/Tools/OctopusDeploy/OctopusDeployArgumentBuilder.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
@agc93 can you remove the unrequired usings?

![image](https://cloud.githubusercontent.com/assets/1271146/16129857/84c23f50-33fe-11e6-9260-37c6dba53bf9.png)